integration_tests/README.md

```diff
@@ -378,6 +378,15 @@ As an example, here is the `spark-submit` command with the cudf_udf parameter on
 $SPARK_HOME/bin/spark-submit --jars "rapids-4-spark_2.12-22.06.0-SNAPSHOT.jar,cudf-22.06.0-SNAPSHOT-cuda11.jar,rapids-4-spark-tests_2.12-22.06.0-SNAPSHOT.jar" --conf spark.rapids.memory.gpu.allocFraction=0.3 --conf spark.rapids.python.memory.gpu.allocFraction=0.3 --conf spark.rapids.python.concurrentPythonWorkers=2 --py-files "rapids-4-spark_2.12-22.06.0-SNAPSHOT.jar" --conf spark.executorEnv.PYTHONPATH="rapids-4-spark_2.12-22.06.0-SNAPSHOT.jar" ./runtests.py --cudf_udf
 ```
 
+### Enabling fuzz tests
+
+Fuzz tests are intended to find more corner cases in testing. We disable them by default because they might randomly fail. 
+The tests can be enabled by appending th option `--fuzz_test` to the command.
```

Comment:
nit: You have "th" instead of "the"

The tests can be enabled by appending the option `--fuzz_test` to the command.

Reply:
Done!